test_on_linux.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+dnu restore
+set -e
+armdir=`pwd`/src/ResourceManagement
+for item in Compute Authorization Graph.RBAC Network Resource Storage WebSite
```

Comment:
i will remove this hard coded list in the netx PR. The goal is to go with auto-discovery